### ISSUE TITLE
Term "loss" in evaluation of a neural network

### ISSUE THREAD

**Issue by jan-christiansen** [sagr4019/ResearchProject#58]
When training and testing a neural network with Keras the tool reports a "loss". How is this value measured?

**Comment by jan-christiansen** [comment]
What is the difference between loss and accuracy?

**Comment by MadeInGer** [comment]
The **loss** output value is the summation of the errors made for each example in training or validation sets.  
The **accuracy** output value will be calculated after learning has been finished. After learning, the validation data will be fed into the model and the number of mistakes the model makes will be kept. Then the percentage of misclassification is calculated.
The wiki has been updated.

**Comment by jan-christiansen** [comment]
Good summary. I have moved the article to the page about [General Terminology](../wiki/general-terminology).
